src/makeInfinityAdder.js

```diff
@@ -1,12 +1,26 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
```

Comment:
Consider adding a comment to explain the purpose of the `adder` function, which is to accumulate the sum and return the result when called without arguments.